test/functional/test_framework/wallet.py

```diff
@@ -182,6 +182,19 @@ def sendrawtransaction(self, *, from_node, tx_hex):
         from_node.sendrawtransaction(tx_hex)
         self.scan_tx(from_node.decoderawtransaction(tx_hex))
 
+    def send_large_transactions(self, node, array_of_large_tx, num_txns, fee_rate):
+        """Creates and sends large transactions with the passed fee rate."""
+        # Create large transactions by appending txouts in vout
+        for _ in range(num_txns):
+            # Create a self transfer here to get the tx details and then append the vout to increase the tx size
+            hex = self.create_self_transfer(from_node=node, fee_rate=fee_rate)['hex']
```

Comment:
Sorry for not noticing this earlier in my last review round, but it looks like the conversion from hex-to-tx is not necessary here, since `create_self_transfer` already provides you a `CTransaction` instance (can be accessed via `['tx']`). I.e. the method can even be shorter:

```diff
--- a/test/functional/test_framework/wallet.py
+++ b/test/functional/test_framework/wallet.py
@@ -187,12 +187,10 @@ class MiniWallet:
         # Create large transactions by appending txouts in vout
         for _ in range(num_txns):
             # Create a self transfer here to get the tx details and then append the vout to increase the tx size
-            hex = self.create_self_transfer(from_node=node, fee_rate=fee_rate)['hex']
-            # Converts it into a CTransaction() instance to append the vouts
-            tx_instance = tx_from_hex(hex)
+            tx = self.create_self_transfer(from_node=node, fee_rate=fee_rate)['tx']
             for txout in array_of_large_tx:
-                tx_instance.vout.append(txout)
-            self.sendrawtransaction(from_node=node, tx_hex=tx_instance.serialize().hex())
+                tx.vout.append(txout)
+            self.sendrawtransaction(from_node=node, tx_hex=tx.serialize().hex())
         return num_txns
```